Add acquisition and analysis scripts

juliacscarvalho opened the pull request:
sysid/python_lib: SysId acquisition and analysis notebooks were added. There is also a script for  for Power Spectral Density and RMS Beam Size plots (fofb_exp_plots).

Commit:
sysid/pythonlib: Add notebook and script to PSD and orbit stability plots

Commit:
sysid/python_lib: Add notebooks for FOFB machine studies acquisition and analysis

juliacscarvalho committed:
Update jupyter notebooks according to apsuite change on saving svd_levels_ampmax

juliacscarvalho committed:
FOFB-acquisition: add acquisition with no PRBS before within the acquisition loop

juliacscarvalho commented:
@ericonr @anacso17

anacso17 commented on sysid/python_lib/FOFB-analysis.ipynb:
```diff
@@ -0,0 +1,385 @@
+    "        prbs[prbs == 0] = -1\n",
+    "        \n",
+    "        acq.params.svd_levels_singmode_idx = data['params']['svd_levels_singmode_idx']\n",
+    "        lvlsx, lvlsy = acq.get_levels_bpms_from_svd(ampmax = acq.params.svd_levels_ampmax*factor, lvl = -val*factor)\n",
```
Should we use here the new prbs data (beam order)?

juliacscarvalho committed:
Update acquisition and analysis notebooks

juliacscarvalho committed:
Add acq_nrpoints_before in data acquisition init

guilhermerc reviewed (changes requested):
Please consider the changes on `sysid/python_lib/FOFB-analysis.ipynb` as we discussed.

juliacscarvalho committed:
Review FOFB analysis notebook

juliacscarvalho committed:
Refactoring orbit RMS functions to be more modular and flexible to available data

ericonr committed:
sysid: update notebook for library update.

ericonr committed:
sysid: update target current.

We now operate with 200mA.

ericonr committed:
sysid: fix saving all BPMs.

We can use the conditional above to save only the relevant BPM, but we
seem to have been having trouble with that.

ericonr committed:
sysid: increase current amplitude to 0.5A.

We are using the 200Hz bandwidth LAMP PI controller, without any
pre-emphasis filter, so we can and should increase the current

ericonr committed:
sysid: increase number of points.

We need 200k samples in order to properly obtain an open loop.
